openedx/features/course_experience/__init__.py

```diff
@@ -79,7 +80,7 @@
 COURSE_ENABLE_UNENROLLED_ACCESS_FLAG = CourseWaffleFlag(SEO_WAFFLE_FLAG_NAMESPACE, 'enable_anonymous_courseware_access')
 
 # Waffle flag to enable relative dates for course content
-RELATIVE_DATES_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'relative_dates')
+RELATIVE_DATES_FLAG = ExperimentWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'relative_dates', experiment_id=17)
```

Comment:
Where does experiment_id come from? Are we documenting that anywhere?

Reply:
Guh, no. I might add a page to the internal Experiments wiki space about this. I had to look at the db to find out what the highest used number was.

Reply:
There are some constants spread throughout platform for it. But I felt like using a constant would just be redundant. The flag name tells you what the 17 is.

Reply:
It might be worth just having a constants file (in the experiments app) that numbers the various experiments, just so that we don't have to go to the DB. On the other hand, maybe that would just give future devs false-confidence that they can use the next number in the constants list (when in fact some optimizely test may have grabbed it).